### ISSUE TITLE
Poll

### ISSUE THREAD

**Issue by teagan75** [GreatOrator/FallCraft-Map#28]
[![](https://api.gh-polls.com/poll/01CEP0W86CF5AFAVJASR8FH0VJ/It%20is%20too%20hard%20to%20get%20from%20point%20A%20to%20B)](https://api.gh-polls.com/poll/01CEP0W86CF5AFAVJASR8FH0VJ/It%20is%20too%20hard%20to%20get%20from%20point%20A%20to%20B/vote)
[![](https://api.gh-polls.com/poll/01CEP0W86CF5AFAVJASR8FH0VJ/It%20is%20too%20easy%20to%20get%20from%20point%20A%20to%20B)](https://api.gh-polls.com/poll/01CEP0W86CF5AFAVJASR8FH0VJ/It%20is%20too%20easy%20to%20get%20from%20point%20A%20to%20B/vote)
[![](https://api.gh-polls.com/poll/01CEP0W86CF5AFAVJASR8FH0VJ/The%20difficulty%20is%20just%20right)](https://api.gh-polls.com/poll/01CEP0W86CF5AFAVJASR8FH0VJ/The%20difficulty%20is%20just%20right/vote)

**Comment by Ladysarajane** [comment]
I had no problem getting from bunker to pleasant valley. Or to find the abandoned mineshaft. As long as the rest of the pack is full of directions, no problem. so just right so far. Only problem is water. I use my bottle caps on buying water. It seems to drain awfully fast. I use 3x as much water as food.

**Comment by teagan75** [comment]
Hmmm I need to tell people there is a water pump in town. Thanks for that info

**Comment by Ladysarajane** [comment]
Good, been saving my bottles at the bankers, now to find that pump and how to fill the bottles.

**Comment by teagan75** [comment]
I believe I put it by the agricraft sprinklers

**Comment by Ladysarajane** [comment]
Had to climb up it to fill up my water bottle, but when I drink it, it doesn't affect my thirst level.  Seems the only thing that affects thirst is "Bottled Water" Filled bottles by the pump say "Water Bottle" and despite how many I drink, my water level remained at 65% until I drank a "Bottled Water"

**Comment by Ladysarajane** [comment]
Yeah I filled it on the wooden agricraft water barrel. Only hand pump I see is the decocraft one by the crops.

**Comment by Ladysarajane** [comment]
I saw that you added them for next update. :)

**Comment by jec0218** [comment]
I love the pack. The only problem i am having is looting the metal shelves and crates from the Decimation mod

**Comment by teagan75** [comment]
That is for multiplayer only

**Comment by GreatOrator** [comment]
https://github.com/GreatOrator/FallCraft-Map/wiki/FAQ

**Comment by jec0218** [comment]
> That is for multiplayer only

then is there a free sever

**Comment by teagan75** [comment]
No. This is a limitation of a mod - Decimation- that is used. There is plenty of loot scattered for the single player.

**Comment by jec0218** [comment]
i am on a LAN with my brothers and we barely have enough loot to go around between the three of us though

**Comment by GreatOrator** [comment]
you could easily set up the strongest of your PC's as an actual server, that would solve that issue.

**Comment by jec0218** [comment]
ok thanks

**Comment by jec0218** [comment]
maybe you should add all of this

> That is for multiplayer only

maybe you should add this to the book

**Comment by jec0218** [comment]
what is the difference between the two maps

**Comment by jec0218** [comment]
also i don't see but only four quests for NORAD, 2 for Colorado springs

**Comment by GreatOrator** [comment]
One is for updates, one is for new maps, the quests open up as you find more/complete other quests

**Comment by jec0218** [comment]
but i am in edit mode and don't see the quests

**Comment by GreatOrator** [comment]
talk to the recorder near elrond in rivendell, he can fix that

**Comment by jec0218** [comment]
> talk to the recorder near elrond in rivendell, he can fix that

i am not in tolkiencraft

**Comment by GreatOrator** [comment]
> > talk to the recorder near elrond in rivendell, he can fix that
> 
> i am not in tolkiencraft

In that case, use bq_admin edit false

**Comment by jec0218** [comment]
Where is  the builder at 0, 0, 0,

**Comment by GreatOrator** [comment]
that was an unfinished quest.

**Comment by teagan75** [comment]
i apologize for that. I neglected to put the coords into that quest. There is a builder in Pleasant Valley as well as the Iron Legion if you want to purchase from them. Otherwise, just ignore the quest.

**Comment by teagan75** [comment]
the robot in the bunker should help you get out of edit mode for future reference

**Comment by PEGAZ19** [comment]
Why can't I search the car and other things? when I right click on it nothing happens
